lib/public/models/BkpRolesSelectionModel.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { SelectionDropdownModel } from '../components/common/selection/dropdown/SelectionDropdownModel.js';
+import { dplDetectorsProvider } from '../services/detectors/dplDetectorsProvider.js';
+import { getRoleForDetector } from '../utilities/getRoleForDetector.js';
+
+const DEFAULT_ROLES_OPTIONS = [{ value: 'admin' }, { value: 'guest' }];
```

Comment:
User is one of them by default. Plus, I suppose one should not be able to pick guest, it should simply be applied if no option is selected

Reply:
User can be both, e.g. me and probably you

Reply:
Are we sure that the `guest` role actually make sense?

Reply:
I don't know, we have it in WebUi Bookkeeping, but at least in case of Bkp, it doesn't serve any meaningful purpose. To be consistent I'd keep it here, but don't insist